modbus_exporter.go

```diff
@@ -45,20 +66,28 @@ func main() {
 	telemetryRegistry.MustRegister(prometheus.NewGoCollector())
 	telemetryRegistry.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
 
+	modbusDurationCounterVec = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "modbus_request_duration_seconds_total",
+		Help: "Total duration of modbus successful requests by target in seconds",
+	}, []string{"target"})
+	telemetryRegistry.MustRegister(modbusDurationCounterVec)
+
+	modbusRequestsCounterVec = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "modbus_requests_total",
+		Help: "Number of modbus request by status and target",
+	}, []string{"target", "status"})
+	telemetryRegistry.MustRegister(modbusRequestsCounterVec)
```

Comment:
This shouldn't be necessary, as these metrics are covered by the `scrape_duration_seconds` and `up` metric provided by Prometheus itself.

Reply:
@superq  What about other monitoring systems that implement openmetrics ,don't have that as internal metrics and just want to scrape the exporter?

Reply:
That's up to them to implement. This is duplicate information.